cozy_management/couchdb.py

```diff
@@ -239,7 +239,7 @@ def get_database_dir():
     import ConfigParser
     result = {}
     file_list = helpers.cmd_exec('find /usr/local/etc/couchdb /etc/couchdb -type f -name "*.ini"')
-    file_list = list(s.rstrip('\n') for s in file_list['stdout'])
+    file_list = list(s for s in file_list['stdout'].rstrip('\n').split('\n'))
```

Comment:
Please avoid single letter variable. It's horrible to maintain (poor meaning and hard to perform renaming)